### PR TITLE
IOTEX-355 Return action hash and block hash with the action core from API

### PR DESCRIPTION
**Pull request by zjshen14**

### CODE REVIEW

File: proto/api/api.proto [hunk 1]
```diff
@@ -105,8 +105,14 @@ message GetActionsByBlockRequest {
   uint64 count = 3;
 }
 
+message Action {
```
Comment (puncsky) [inline proto/api/api.proto]: it feels strange to access iotextypes.Action  like getActionsRespons.actions[n].action

I suggest naming it `MintedAction` or `ActionResult` ... or other kinds of action you can name it

**Comment by Frankonly** [comment]
ioctl is ok.

**Comment by zjshen14** [comment]
Updated per comments

File: api/api.go [hunk 2]
```diff
@@ -411,31 +411,35 @@ func (api *Server) getActions(start uint64, count uint64) (*iotexapi.GetActionsR
...
+			}
+			res = append(res, act)
 		}
 	}
 
-	return &iotexapi.GetActionsResponse{Actions: res}, nil
+	return &iotexapi.GetActionsResponse{ActionInfo: res}, nil
 }
 
 // getAction returns action by action hash
```
Comment (raullenchai) [inline api/api.go]: // getSingleAction

**Pull request merged**